src/index.js

```diff
@@ -0,0 +1,182 @@
+"use strict";
+
+const state = {
+    temp : 80,
+    tempColor: "red",
+    changeCity: "Orlando",
+    skyCondition: document.getElementById("tempContainer"),
+    lat: 28.5384,
+    lon: 81.3789,
+};
+
+const temp = document.getElementById("tempValue");
+temp.textContent = state.temp + "°";
+temp.style.color = state.tempColor;
+
+
+const increaseTemp = () => {
+    state.temp++;
+    const increase = document.getElementById("tempValue");
+    increase.textContent = `${state.temp}°`;
+};
+
+const decreaseTemp = () => {
+    state.temp--;
+    const decrease = document.getElementById("tempValue");
+    decrease.textContent = `${state.temp}°`;
+};
+
+const tempColor = () => {
+    if(state.temp <= 49){
+        state.tempColor = "teal";
+        temp.style.color = state.tempColor;
+    } else if(50 <= state.temp && state.temp <= 59){
+        state.tempColor = "green"
+        temp.style.color = state.tempColor;
+    } else if(60 <= state.temp && state.temp <= 69){
+        state.tempColor = "yellow";
+        temp.style.color = state.tempColor;
+    } else if(70 <= state.temp && state.temp <= 79){
+        state.tempColor = "orange";
+        temp.style.color = state.tempColor;
+    } else if(state.temp >= 80){
+        state.tempColor = "red";
+        temp.style.color = state.tempColor;
+    }
+};
+
+const tempImage = () => {
```

Comment:
Could rename this method to `setTempImage` for clarity about what the method does.